modules/microshift-rhoai-override-kserve-config.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-rhoai-override-kserve-config_{context}"]
+= Overriding KServe configuration
+
+If you want to override KServe settings to customize your model serving environment, you can follow the general steps for your operating system.
+
+Option 1::
+
+. Make a copy of the existing `ConfigMap` file, `inferenceservice-config`, in the `redhat-ods-applications` namespace.
+
+. Edit the settings you want to change.
+
+. Overwrite the existing `ConfigMap` object.
+
+. Restart KServe by either by deleting the pod or scaling the `Deployment` down to 0 and then back up to 1.
+
+Option 2::
+
+. Copy the `ConfigMap` file, `/usr/lib/microshift/manifests.d/001-microshift-ai-model-serving/kserve/inferenceservice-config-microshift-patch.yaml`.
+
+. Edit the settings you want to change.
+
+. Apply the `ConfigMap` object.
+
+. Restart KServe by either by deleting the pod or scaling the `Deployment` down to 0 and then back up to 1.
+
+For {op-system-ostree} and {op-system-image} systems::
+
+. Create a new manifest.
+
+. Ensure that the new manifest is applied after the `/usr/lib/microshift/manifests.d/001-microshift-ai-model-serving` file.
```

Comment:
Just FYI, I'm gonna change the number to `010`. So we can recommend using prefix 011
In the PR that will introduce the split of kserve and runtimes manifests, kserve will be 010, manifests will be 050.
We might also instruct that the manifest must be in the /usr/lib/microshift/manifests.d/, because if it's in the /etc/microshift/manifests.d, then it'll be too late (first all manifests from /usr/... are applied, then /etc/...)

Reply:
Ah, forgot: I'm gonna also rename the manifest to `/usr/lib/microshift/manifests.d/010-microshift-ai-model-serving-kserve`